packages/wonder-stuff-server/src/start-server.js

```diff
@@ -0,0 +1,193 @@
+// @flow
```

Comment:
Copied from render-gateway but updated to remove Google-specifics and to use `Runtime` enum as well as updated `ServerOptions` (was `GatewayOptions`) to allow for non-Google-ness